### PR TITLE
add oci-notify utility

### PR DESCRIPTION
**Pull request by guidotijskens**
updated oci-notify code with suggestions from Frank Deng and Joe Kennedy, limit the length of the subject, limit the size of the body to 10 chunks of 64KB; looks for location of the private key in the config file iso hardcode directory
.
run on OL7 and OL8

### CODE REVIEW

File: man/man1/oci-notify.1 [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+.B oci-notify
+utility sends a message to an
+.B Oracle Cloud Infrastructure
+notification service. A message is composed of a title and file. Large files
+are split into 64KB chunks. It is the responsibility of the notification service
+configuration where and how the messages are delivered.
+
+.B oci-notify
+command writes log and error messages to the logfile
+.B /var/log/oci-notify.log
```
Comment (ejannett) [inline man/man1/oci-notify.1:32]: What about having a common approach for all oci-utils utilities ? is it debug messages or messages useful for end users ?

File: man/man1/oci-notify.1 [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+.B oci-cli
+installed. Authentication is attempted via
+.B oci-sdk
+configuration, direct authentication. On failure, the instance principal authentication is used.
+
+.B oci-notify
+expects the configuration files by default in
+.B /etc/oci-utils
+but this is overruled by the environment variable
+.B OCI_CONFIG_DIR.
```
Comment (ejannett) [inline man/man1/oci-notify.1:47]: Is it a requirement ? This is not aligned with other oci-utils commands

**Review by ejannett — COMMENTED** [review]
Hi,
Why not having used the OCI sdk directly rather than writing a wrapper around 'oci' cli ?

**Comment by guidotijskens** [comment]
manu, this is a quick adaptation of the al-notify bash script; at the time of the request there was time pressure; now it will not be released very soon, will be ported to python and using sdk;

**Review by samiam69 — APPROVED** [review]

**Pull request closed without merging**